diff --git a/third_party/liw/liw/include/liw/liw_time.h b/third_party/liw/liw/include/liw/liw_time.h
index 21248e7..6bb388c 100644
--- a/third_party/liw/liw/include/liw/liw_time.h
+++ b/third_party/liw/liw/include/liw/liw_time.h
@@ -34,7 +34,7 @@ __BEGIN_DECLS
 int liwSleep(long int sec, long int nsec);
 
 /* CPU clock using RDTSC */
-extern inline long long liwClock(void){
+inline long long liwClock(void){
   unsigned long long ret;
   __asm__ volatile ( "rdtsc" : "=A" (ret) );
   return ret;
